<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EssentialParaEdit</class>
 <widget class="QDialog" name="EssentialParaEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Essential parameters</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>589</width>
     <height>377</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="13" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Gravity</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Ambient light</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Background color</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Point light</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <layout class="QFormLayout" name="formLayout_17">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_12">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_12">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QFormLayout" name="formLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_4">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_4"/>
      </item>
     </layout>
    </item>
    <item row="3" column="3">
     <layout class="QFormLayout" name="formLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_6">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_6">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="value">
         <double>-1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Collision damping</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Edit color</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <layout class="QFormLayout" name="formLayout_13">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_8">
        <property name="text">
         <string>v</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_8">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Refractive index</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Edit color</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <layout class="QFormLayout" name="formLayout_11">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_7">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_7">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="nearClipLabel">
        <property name="text">
         <string>near clip</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="nearClipDoubleSpinBox">
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <layout class="QFormLayout" name="formLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_3">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_3">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="value">
         <double>22.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Edit color</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fovLabel">
        <property name="text">
         <string>fov</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="fovDoubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.785000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="farClipDoubleSpinBox">
        <property name="maximum">
         <double>400.000000000000000</double>
        </property>
        <property name="value">
         <double>200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="farClipLabel">
        <property name="text">
         <string>far clip</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="2">
     <layout class="QFormLayout" name="formLayout_15">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_10">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_10">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <layout class="QFormLayout" name="formLayout_18">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_13">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_13">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>-9.800000000000001</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <layout class="QFormLayout" name="formLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_2">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_2">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="value">
         <double>6.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="2">
     <layout class="QFormLayout" name="formLayout_14">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_9">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_9">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="3">
     <layout class="QFormLayout" name="formLayout_16">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_11">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_11">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="value">
         <double>12.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Orientation</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <layout class="QFormLayout" name="formLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_5">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_5">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="13" column="3">
     <layout class="QFormLayout" name="formLayout_19">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_14">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_14">
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="14" column="2" colspan="2">
     <layout class="QFormLayout" name="formLayout_20">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel_15">
        <property name="text">
         <string>v</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox_15">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EssentialParaEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EssentialParaEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
